compiler/src/parsing/ast_helper.re

```diff
@@ -192,8 +196,47 @@ module Exp = {
     mk(~loc?, ~attributes?, PExpConstant(a));
   let tuple = (~loc=?, ~attributes=?, a) =>
     mk(~loc?, ~attributes?, PExpTuple(a));
-  let record = (~loc=?, ~attributes=?, a) =>
-    mk(~loc?, ~attributes?, PExpRecord(a));
+  let record = (~loc=?, ~attributes=?, a, b) =>
+    mk(~loc?, ~attributes?, PExpRecord(a, b));
+  let record_unprocessed = (~loc=?, ~attributes=?, a) =>
```

Comment:
The names of everything in this module are pretty semantic, so I'd probably fold this and `record` into one function just called `record`.

Reply:
I created a separate `record_unprocessed` function here because in `ast_mapper` a call to `record` is made as well, and I felt it might be a bit wasteful to do all of this preprocessing at `ast_mapper` when `PExpRecord` would have already been preprocessed at that point. If you think this isn't that big of a deal then I could remove it though.

Reply:
Another idea that would make the names more semantic: how about still keeping these functions separate but changing the name of `record_unprocessed` to `record_fields` as this function essentially processes the record fields into a processed `record`

Reply:
I think that makes sense.